library/tabs/src/TabsWrapper.php

```diff
@@ -60,14 +60,27 @@ public function genCss()
         if (!defined('INCLUDED_TW_ONETIME_CSS')) {
             define(INCLUDED_TW_ONETIME_CSS, true);
             $s .= <<<EOD
-.ui-tabs .ui-tabs-panel {
-  padding: 0px 0px;
-}
-.ui-tabs .ui-tabs-nav .ui-tabs-anchor {
-  padding: .1em .4em;
-}
-EOD;
+        #enctabs-wrapper{
```

Comment:
Actually the TabsWrapper class should not have anything that is specific to just one (enctabs) instance of it. I think this style rule belongs in encounter_top.php until a better place is found.

Reply:
Done.